src/category_theory/natural_transformation.lean

```diff
@@ -36,6 +36,9 @@ restate_axiom nat_trans.naturality'
 -- components of natural transfomations moving earlier.
 attribute [simp, reassoc] nat_trans.naturality
 
+lemma congr_app {F G : C ⥤ D} {α β : nat_trans F G} (h : α = β) (X : C) : α.app X = β.app X :=
```

Comment:
Don't you want to use the `\hom`-arrow instead of `nat_trans`?

Reply:
In this file it actually doesn't exist yet! We've just defined natural transformations, but haven't yet put a category structure on functors.